schema/pgsql/schema.sql

```diff
@@ -141,10 +185,13 @@ CREATE TABLE rotation_member (
     UNIQUE (rotation_id, contact_id),
     UNIQUE (rotation_id, contactgroup_id),
     CHECK (num_nonnulls(contact_id, contactgroup_id) = 1),
+    CHECK (deleted = 'y' OR position IS NOT NULL),
```

Comment:
This still creates constraints named `rotation_member_check` and `rotation_member_check1`, so better name them here as well.